### ISSUE TITLE
TypeError: mean() got an unexpected keyword argument 'level'

### ISSUE THREAD

**Issue by ramGoli** [amarvin/fantasy-football-bot#75]
Running into this error after running ffbot.scrape

```
/opt/homebrew/lib/python3.11/site-packages/ffbot/scraper.py:126: FutureWarning: Passing literal html to 'read_html' is deprecated and will be removed in a future version. To read from a literal string, wrap it in a 'StringIO' object.
  df2 = pd.read_html(html)[0]
Scraping weekly forecasts: 100%|████████████████████████████████████████████████████████████████████████████████████████████| 1121/1121 [03:24<00:00,  5.49it/s]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/homebrew/lib/python3.11/site-packages/ffbot/scraper.py", line 160, in scrape
    means = available.groupby(["Position"])["Remaining"].nlargest(3).mean(level=0)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/pandas/core/series.py", line 6226, in mean
    return NDFrame.mean(self, axis, skipna, numeric_only, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/pandas/core/generic.py", line 11969, in mean
    return self._stat_function(
           ^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/pandas/core/generic.py", line 11922, in _stat_function
    nv.validate_func(name, (), kwargs)
  File "/opt/homebrew/lib/python3.11/site-packages/pandas/compat/numpy/function.py", line 416, in validate_func
    return validation_func(args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/pandas/compat/numpy/function.py", line 88, in __call__
    validate_args_and_kwargs(
  File "/opt/homebrew/lib/python3.11/site-packages/pandas/util/_validators.py", line 223, in validate_args_and_kwargs
    validate_kwargs(fname, kwargs, compat_args)
  File "/opt/homebrew/lib/python3.11/site-packages/pandas/util/_validators.py", line 164, in validate_kwargs
    _check_for_invalid_keys(fname, kwargs, compat_args)
  File "/opt/homebrew/lib/python3.11/site-packages/pandas/util/_validators.py", line 138, in _check_for_invalid_keys
    raise TypeError(f"{fname}() got an unexpected keyword argument '{bad_arg}'")
TypeError: mean() got an unexpected keyword argument 'level'
>>> 
```

**Comment by amarvin** [comment]
Thanks for reporting this. I test `ffbot` in Python 3.8, 3.9 and 3.10, but hadn't previously tested in Python 3.11. There's then a change in the latest `pandas` for Python 3.11 that removed the `level` kwarg from `mean()`. I'll make a fix for you!

**Comment by amarvin** [comment]
I've published a new version of `ffbot` with a fix for you that you can install using `pip install -U ffbot`. If it doesn't work, please reopen this issue.

**Comment by ramGoli** [comment]
Thank you for the fix. No errors now!

**Comment by ramGoli** [comment]
When is comes to NFL, I am relatively in experienced. I tried understanding what "Discounted Points" is, and I don't get it, I think I understand VOR (ELI5: If positive value, then good, if negative, then bad??) 

Just commenting here to understand discounted points and I don't find any online search results for "discounted points NFL"

**Comment by amarvin** [comment]
Ah, good question. It's not a standard NFL or Fantasy NFL term, and I just created the term for `ffbot`. The idea is that when optimizing your players using `ffbot`, it tries to maximize your forecasted discounted points for all future weeks. Those points are "discounted" based on time, so that this week is more important for the optimizer than like 10 weeks from now. You see the same "discounted" term used in financial optimization due to the time-value of money.
